### PR TITLE
Updated the message headers sample to include V6

### PR DESCRIPTION
**Pull request by SzymonPobiega**
* Updated the sample to include V6
* Split header page into two parts
* Updated the references

### CODE REVIEW

**Comment by SzymonPobiega** [comment]
@SimonCropp please review.

File: nservicebus/messaging/manipulating-message-headers.md [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+summary: Access and manipulate the built in NServiceBus headers or add custom headers.
+tags: 
+- Header
+redirects:
+- nservicebus/how-do-i-get-technical-information-about-a-message
+related:
+- samples/header-manipulation
+---
+
+Extra information about a message is communicated over the transport as secondary information to the message body. The mechanism for header communication is either native headers, if the transport supports that feature, or via a serialized collection of key value pairs. Message headers are very similar, in both implementation and usage, to http headers. NServiceBus uses them internally to manage messaging patters, eg pub/sub and request/response, and users of NServiceBus can use headers to implement a variety of features.
```
Comment (SimonCropp) [inline nservicebus/messaging/manipulating-message-headers.md]: i think u meant to remove the first part of this file as part of the split?

Comment (SzymonPobiega) [inline nservicebus/messaging/manipulating-message-headers.md]: Forgot to save the file in MP :\

**Comment by SzymonPobiega** [comment]
@SimonCropp added the missing changes.

File: samples/header-manipulation/Version_6/Sample/app.config [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
```
Comment (SimonCropp) [inline samples/header-manipulation/Version_6/Sample/app.config]: redundant file

**Pull request merged**